app/views/ssh_keys/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_for @ssh_key, :url => user_ssh_keys_path(:user_id => @user) do |f| %>
+  <%= base_errors_for @ssh_key %>
+  <% if @ssh_key.errors.include?(:fingerprint) || @ssh_key.errors.include?(:length) %>
+    <p><%= _('Please correct the error(s) below and submit your changes again.') %></p>
+
+    <div class="alert alert-warning alert-block base">
+      <% [:fingerprint, :length].each do |field|%>
+        <% @ssh_key.errors.full_messages_for(field).each do |e| %>
+          <li><%= e %></li>
+        <% end %>
+      <% end %>
+    </div>
+  <% end %>
+  <%= textarea_f f, :key, :size => "col-md-8" , :rows=> '3' %>
```

Comment:
should we allow to upload ssh key as well?
also, i could see us using the ACE editor to display / edit the key.
for example, again from openshift:
![image](https://cloud.githubusercontent.com/assets/20167/23160586/bb6ff22c-f82f-11e6-8a1e-d6c365da5c4d.png)